Fix Download exception when kodi isn't playing

burekas7 opened the pull request:
- Fix the error exception from kodi when downloading subtitle is done
  when kodi isn't playing (Added an if conditition in mirror_sub function)

Note: When the user will press on a subtitle, the subtitle will still be
downloaded to the temp folder in the addon folder in userdata (As it
done currently for normal downloading when kodi is playing), after that
the search subtitles dialog will be closed. But this is not should be
relevant to the user for the main purpose of using it only for watching
the subtitles result list.

In Addition:
- Set a new local variable **isKodiPlaying** instead of writing
  xbmc.Player().isPlaying() on each place.
- Fix speeling "Serach for" to "Search for..."

burekas7 committed:
Fix Download exception when kodi isn't playing

* Fix the error exception from kodi when downloading subtitle is done
when kodi isn't playing (Added an if conditition in mirror_sub function)
Note: When the user will press on a subtitle, the subtitle will still be
downloaded to the temp folder in the addon folder in userdata (As it
done currently for normal downloading when kodi is playing), after that
the search subtitles dialog will be closed. But this is not should be
relevant to the user for the main purpose of using it only for watching
the subtitles result list.

In Addition:
* Set a new local variable __isKodiPlaying__ instead of writing
xbmc.Player().isPlaying() on each place.
* Fix speeling "Serach for" to "Search for..."

felixmosh commented:
Your PR has a conflict...
Anyway, i will put the right IF in the right place...looks like you put the if only on mirror_subs...

burekas7 commented:
What does it mean when the PR has a conflict?
Anyway, check my file following my description and do the changes (3 changes, the "if" in the mirror_subs, the variable, and the string)
Thanks.

burekas7 committed:
Merge remote-tracking branch 'refs/remotes/XBMCil/master' into subscenter_burekas_downloadFix

burekas7 committed:
Fix Download exception when kodi isn't playing

* Fix the error exception from kodi when downloading subtitle is done
when kodi isn't playing (Added an if conditition in mirror_sub function)
* Note: When the user will press on a subtitle, the subtitle will still
be downloaded to the temp folder in the addon folder in userdata (As it
done currently for normal downloading when kodi is playing), after that
the search subtitles dialog will be closed. But this is not should be
relevant to the user for the main purpose of using it only for watching
the subtitles result list.

In Addition:
* Set a new local variable __isKodiPlaying__ instead of writing
xbmc.Player().isPlaying() on each place.
* Fix speeling "Serach for" to "Search for..."

burekas7 commented:
The conflict was because I didn't update my file from the master before doing the fixes.
I made a new PR #9  with no conflicts with the if condition and the new local variable.

burekas7 committed:
Revert "Fix Download exception when kodi isn't playing"

This reverts commit 060d0e8604bbd94aef25b11d59e10048c044d327.

burekas7 committed:
Merge pull request #2 from burekas7/subscenter_burekas

Merge download fix branch to main branch with all the changes

burekas7 committed:
Merge pull request #3 from burekas7/subscenter_burekas_downloadFix

Merge download fix branch to main branch with all the changes

CaTzil commented:
Committed cleaner variation...
